xbmc/utils/ActorProtocol.cpp

```diff
@@ -117,7 +117,7 @@ void Protocol::ReturnMessage(Message *msg)
   freeMessageQueue.push(msg);
 }
 
-bool Protocol::SendOutMessage(int signal, void *data /* = NULL */, size_t size /* = 0 */, Message *outMsg /* = NULL */)
+bool Protocol::SendOutMessage(int signal, const void *data /* = NULL */, size_t size /* = 0 */, Message *outMsg /* = NULL */)
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types